UPGRADE-4.x.md

```diff
@@ -1,13 +1,30 @@
 UPGRADE 4.x
 ===========
 
-UPGRADE FROM 3.x to 3.x
+UPGRADE FROM 4.x to 4.x
 =======================
 
 ### Support for NelmioApiDocBundle > 3.6 is added
 
 Controllers for NelmioApiDocBundle v2 were moved under `Sonata\UserBundle\Controller\Api\Legacy\` namespace and controllers for NelmioApiDocBundle v3 were added as replacement. If you extend them, you must ensure they are using the corresponding inheritance.
 
+### Fix REST API routing paths
+
+In `SonataUserBundle` v4.6.0 some wrong REST API routing paths was added. You should avoid using it to keep API doc cleaner. Change this one to correct paths:
+   - `/group.{_format}` => `/groups.{_format}`
+   - `/group/{id}.{_format}` => `/groups/{id}.{_format}`
+   - `/user.{_format}` => `/users.{_format}`
+   - `/user/{id}.{_format}` => `/users/{id}.{_format}`
+   - `/user/{userId}/{groupId}.{_format}` => `/users/{userId}/groups/{groupId}.{_format}`
+
+  Then change your routing to:
+
+      sonata_api_user:
+          prefix: /api/user
+          resource: "@SonataUserBundle/Resources/config/routing/fix_api.xml"
```

Comment:
IMO, we should not use "fix" in these routing files. I think we could use "api_plural_model" or something like that.

Reply:
standard_api.xml?

Reply:
Or default_api.xml?

Reply:
base_api, legacy_paths_api

Reply:
I'm fine with any of these proposals :+1:

Reply:
standard will be the best.  It mean it is standard api which should be used.